browser/css/jsdialogs.css

```diff
@@ -1725,6 +1725,9 @@ input[type='number']:hover::-webkit-outer-spin-button {
 	justify-content: center;
 }
 
+#SparklineDialog .jsdialog.menubutton img {
```

Comment:
I have impression it's the general problem with these menubuttons -> examples in sidebar

could you check? maybe it is good to not target only the sparkline dialog in this case